src/components/PostDialog.tsx

```diff
@@ -20,15 +26,35 @@ export const PostDialog: Component<{
   };
 
   return (
-    <dialog ref={dialogRef} onCancel={cancelHandler}>
-      <div>
-        <Logo
-          src={props.frontmatter.logo.src}
-          alt={props.frontmatter.logoAlt}
-        ></Logo>
-        <h2>{props.frontmatter.title}</h2>
-        <h3>{props.frontmatter.company}</h3>
+    <dialog ref={dialogRef} onCancel={cancelHandler} onClick={cancelHandler}>
```

Comment:
Consider breaking this down into smaller chunks for readability. The extra spans and divs are making it kind of hard to read